app/mixins/custom-form.js

```diff
@@ -16,96 +16,96 @@ export default Mixin.create(MutableArray, {
         isIncluded      : false,
         event           : parent
       }),
       this.store.createRecord('custom-form', {
-        fieldIdentifier : 'Short abstract',
+        fieldIdentifier : 'shortAbstract',
         form            : 'session',
         type            : 'text',
         isRequired      : false,
         isIncluded      : true,
         event           : parent
       }),
       this.store.createRecord('custom-form', {
-        fieldIdentifier : 'Long abstract',
+        fieldIdentifier : 'longAbstract',
         form            : 'session',
         type            : 'text',
         isRequired      : false,
         isIncluded      : false,
         event           : parent
       }),
       this.store.createRecord('custom-form', {
-        fieldIdentifier : 'Comments',
+        fieldIdentifier : 'comments',
         form            : 'session',
         type            : 'text',
         isRequired      : false,
         isIncluded      : true,
         event           : parent
       }),
       this.store.createRecord('custom-form', {
-        fieldIdentifier : 'Track',
+        fieldIdentifier : 'track',
         form            : 'session',
-        type            : 'text',
+        type            : 'select',
         isRequired      : true,
         isIncluded      : true,
         isFixed         : true,
         event           : parent
       }),
       this.store.createRecord('custom-form', {
-        fieldIdentifier : 'Session type',
+        fieldIdentifier : 'sessionType',
         form            : 'session',
-        type            : 'text',
+        type            : 'select',
         isRequired      : false,
         isIncluded      : true,
         event           : parent
       }),
       this.store.createRecord('custom-form', {
-        fieldIdentifier : 'Level',
+        fieldIdentifier : 'level',
         form            : 'session',
         type            : 'text',
         isRequired      : false,
         isIncluded      : false,
         event           : parent
       }),
       this.store.createRecord('custom-form', {
-        fieldIdentifier : 'Language',
+        fieldIdentifier : 'language',
         form            : 'session',
-        type            : 'text',
+        type            : 'select',
         isRequired      : false,
         isIncluded      : false,
         event           : parent
       }),
       this.store.createRecord('custom-form', {
-        fieldIdentifier : 'Slides',
+        fieldIdentifier : 'slidesUrl',
         form            : 'session',
-        type            : 'text',
+        type            : 'file',
         isRequired      : false,
         isIncluded      : true,
         event           : parent
       }),
       this.store.createRecord('custom-form', {
-        fieldIdentifier : 'Video',
+        fieldIdentifier : 'videoUrl',
         form            : 'session',
-        type            : 'text',
+        type            : 'file',
```

Comment:
@niranjan94 The server only supports `file` that's why I have used it instead of file or url, we can take care of this in the component for file upload itself.